src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " start work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getNameMachine() + " stoped work");
```

Comment:
Similarly, the message in the 'stopWork' method should be more informative and indicate that an 'Excavator' has stopped working. Also, there is a typo in 'stoped', which should be 'stopped'. Consider changing the message to: 'Excavator ' + getNameMachine() + ' stopped working.'